src/org/zaproxy/zap/extension/AllInOneNotes/EventConsumerImpl.java

```diff
@@ -0,0 +1,87 @@
+package org.zaproxy.zap.extension.AllInOneNotes;
```

Comment:
This shoukd have a license too, all classes should 😊

Reply:
The package should be all lower case.

Reply:
thanks, both issues addressed!